code/datums/diseases/advance/symptoms/goat.dm

```diff
@@ -0,0 +1,124 @@
+/datum/symptom/goat
+	name = "Caprinae Muscular Hypertrophy"
+	desc = "This viral infection causes the afflicted to experience a rapid growth in leg muscles, may cause increased aggression and a desire to climb."
+	//Stats will be based off of Necropolis Seed because this symptom is powerful and should be slow
+	stealth = -1
+	resistance = 3
+	stage_speed = -8 			//Threshold will be based off this, so it needs to start low.
+	transmission = -4 			//Equal to the hardest hitting symptoms. Not possible to make airborne
+	level = 9 					//Powerful symptom, should be rare(ground sample only) as well as one of the few positive level 9s.
+	severity = -1 				//Positive with a catch.
+	symptom_delay_min = 15
+	symptom_delay_max = 20
+	var/speed_bonus = -0.10 	//Negative values increase speed, positive decreases.
+	var/has_headbutt = FALSE
+	threshold_desc = "<b>Resistance 6: </b>The infected becomes slightly quicker on their feet.<br>\
+					<b>Stage Speed 10: </b>The legs of the infected contain enough muscle mass to leap forward in a charge."
+
+/datum/symptom/goat/severityset(datum/disease/advance/A)
+	. = ..()
+	if((A.resistance >= 6) || (A.stage_rate >= 10))
+		severity -= 2
+
+/datum/symptom/goat/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	RegisterSignal(A.affected_mob, COMSIG_MOB_SAY, .proc/handle_speech) //Causes the infected to 'roll' their A's
+	if(A.resistance >= 6)
+		speed_bonus = -0.2 //While fairly low, consider triple-stacked viruses. That is 60% possible speed.
+	if(A.stage_rate >= 10)
+		has_headbutt = TRUE
+	
+/datum/symptom/goat/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	
+	if(A.stage >= 2)
+		if(prob(10)) //Works as both a virus 'warning' and hints at effects of the virus itself.
+			to_chat(M, "<span class='warning'>[pick(
+			"You feel like chewing on some cans.",
+			"For some reason, you want to climb to high places.", 
+			"Vines are no match for you.", 
+			"You're starting to understand why Pete is so mad.",
+			"You have a desire to headbutt someone.", 
+			"You let out a bleat under your breath.")]</span>")
+			
+	if((A.stage >= 3) && (speed_bonus != FALSE) && (ishuman(M))) //Grants speed, trash eating trait and climb speed bonus.

```

Comment:
```suggestion
	if(A.stage >= 3 && speed_bonus && ishuman(M)) //Grants speed, trash eating trait and climb speed bonus.
```
Were you supposed to check if speed bonus was false or true?

Reply:
The speed bonus is given a value before that section, which is overwritten by making it FALSE afterwards.
It ensures that you only get it that one time.
Which on a second check, I have the speed boost given after it's made false so it does nothing.
Got to flip that

Reply:
Alright, go ahead and commit this. Also, there is a commit to batch option that lets you commit all of this under one commit instead of making 14 commits for very small changes.

Reply:
Fixed that section.
Where is the batch commit option?
Haven't been able to find any way to merge smaller changes like that.

Reply:
It should be next to the commit button on the website,you do have to go to the files changed section iirc